### PR TITLE
homepage 4-10 content

### LINKED ISSUE #190: Create homepage content 4-10 layout
April 10th homepage content html fragment.

### PR DESCRIPTION
**Pull request by karigaron17**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #190

Test URLs:
- Before: https://main--maidenform--hlxsites.hlx.live/
- After: https://190-create-homepage-content-4-10-layout--maidenform--hlxsites.hlx.live/

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -90,118 +173,6 @@
   size-adjust: 106.71%;
 }
 
-/* fallback fonts for Zarah font family */
-
-@font-face {
```
Comment (herzog31) [inline styles/styles.css:95]: Is there a reason you removed the fallback fonts? Font files should be loaded delayed via `lazy-styles.css` so they don't block the initial page rendering. If you need to add additional fallback fonts, you can leverage https://github.com/pixel-point/fontpie.

When using fonts in your styles, please use the variables we defined, e.g. `font-family: var(--ff-zahrah-semibold)`.

File: fragments/2023_04/2023-04-10.html [hunk 2]
```diff
@@ -0,0 +1,917 @@
...
+  .mdf-hp-topBanner span {
+    font-size: clamp(1.3rem, 3.6vw, 2.2rem);
+    display: inline-block;
+    text-transform: uppercase;
+  }
+
+  /* hero ----------------------------------------------------------*/
+  @media (max-width: 599px) {
+    .mdf-hp-hero {
+      background: url('//cdn.maidenform.com/wysiwyg/MDF/Franklin/2023_04_10/040623_MF_HP_MB_Hero_03b.jpg?trim=620,0,0,0') 0 85%/ 100% 53% no-repeat;
```
Comment (herzog31) [inline fragments/2023_04/2023-04-10.html:199]: Please ensure all images are delivered either via Franklin directly or served from the same host (franklin.maidenform.com). I recommend to upload the images to Sharepoint and publish them via the Sidekick.

File: fragments/2023_04/2023-04-10.html [hunk 3]
```diff
@@ -0,0 +1,917 @@
...
+    .mdf-hp-quick-change-bra .inner-wrapper::after {
+      width: 13%;
+      top: 81%;
+      left: 57%;
+    }
+  }
+</style>
+
+<div class="homepage-wrapper">
+  <!-- Top Banner -->
```
Comment (herzog31) [inline fragments/2023_04/2023-04-10.html:684]: All of these are very good candidates for blocks btw.

File: styles/styles.css (hunk 1 above)
Comment (karigaron17) [reply styles/styles.css:95]: The fallback was to Arial, which is a san-serif font and not at all like the Zahrah font. We can just use the font-family: font, serif for a fall back for now. We be shifting all the fonts soon and removing both Gilroy and Zahrah all together.

File: fragments/2023_04/2023-04-10.html (hunk 3 above)
Comment (karigaron17) [reply fragments/2023_04/2023-04-10.html:684]: We will slowly start moving the repeatable parts to blocks as we can working with the designers. So much of our content is one time layout specific and not the same layout again.

File: styles/styles.css (hunk 1 above)
Comment (herzog31) [reply styles/styles.css:95]: Good point. We can adjust the fallback fonts to use a serif font like Times for Zahrah. To optimize performance, we should (for now) load font files only in `lazy-styles.css` however and do it in a way it doesn't cause layout shift. That's why we use the linked tool to generate a fallback font that creates the same dimensions as Zahrah.

> The font variable did not seem to carry over and work in the fragment file either.
We should check this. The variables are defined globally, so they should be available everywhere on the page.

Comment (karigaron17) [reply styles/styles.css:95]: I am working on reverting this back to the lazy-styles and a fallback with a serif font. the only place the variable didn't seem to work was in the fragment html <style> tag file. I will try it again with the lazy-style updated.

**Comment by karigaron17** [comment]
@herzog31 I believe the fonts and image comments should all be resolved now. I updated the Zahrah fallback fonts and added them back in and the fonts are loading correct now. Moved the images to SharePoint and updated all the src urls to for them.

I will look at splitting a few elements out to blocks for the next build, but will keep some of the css part of the html fragment as the color will need to be custom per update.

looks like there were other updates to the repo that when synced are showing as errors in the PR now.

File: styles/styles.css [hunk 4]
```diff
@@ -223,24 +224,24 @@
...
-  --ff-zahrah-bold-italic: 'Zahrah Bold Italic 700', 'Zahrah Bold Italic 700 Fallback', serif;
-  --ff-zahrah-semibold: 'Zahrah Semibold 600', 'Zahrah Semibold 600 Fallback', serif;
-  --ff-zahrah-semibold-italic: 'Zahrah Semibold Italic 600', 'Zahrah Semibold Italic 600 Fallback', serif;
-  --ff-zahrah-medium: 'Zahrah Medium 500', 'Zahrah Medium 500 Fallback', serif;
-  --ff-zahrah-medium-italic: 'Zahrah Medium Italic 500', 'Zahrah Medium Italic 500 Fallback', serif;
-  --ff-zahrah-regular: 'Zahrah Regular 400', 'Zahrah Regular 400 Fallback', serif;
-  --ff-zahrah-regular-italic: 'Zahrah Regular Italic 400', 'Zahrah Regular Italic 400 Fallback', serif;
-  --ff-zahrah-light-italic: 'Zahrah Light Italic 300', 'Zahrah Light Italic 300 Fallback', serif;
-  --ff-zahrah-light: 'Zahrah Light 300', 'Zahrah Light 300 Fallback', serif;
+  --ff-zahrah-bold: 'Zahrah Bold', serif;
```
Comment (herzog31) [inline styles/styles.css:235]:
```suggestion
  --ff-zahrah-bold: 'Zahrah Bold', 'Zahrah Bold Fallback', serif;
```
You forgot about all fallback fonts in the Zahrah font variables.

**Review by herzog31 — APPROVED** [review]
Looks good to merge 👍

**Pull request merged**